README.md

```diff
@@ -66,9 +66,9 @@ cp .env.example .env
 
 7. Construct the database.
 
-#### All users
+### Construct the database
```

Comment:
This isn't needed for team members. I was trying to differentiate between "All users" and "Infrastructure team members only".

Reply:
The meaning of "All users" wasn't clear to me. Whether that's referring to accounts on the site, or some other step that I may or may not want to follow. I know what it means of course, but I'm trying to read it from the next person's POV. I think team members know what this does and will know to skip it.

I did add phrasing the infra team section to describe that as something that can be done "instead".

Feel free to change this back if that's clearer to you. Was being bold, and lazy as I haven't figured out the GitHub suggestion protocol yet. Sorry about that.

```suggestion
### All users
```